webapp/src/components/router_showcase/router_showcase.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+    Switch,
+    Route,
+    useLocation,
+} from 'react-router-dom';
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+import {useSelector} from 'react-redux';
+
+import {id} from 'src/manifest';
+
+export default function RouterShowcase() {
+    const currentTeam = useSelector(getCurrentTeam);
+    const query = useQuery();
+
+    function pushTeamUrlPath(path) {
+        return window.WebappUtils.browserHistory.push(teamUrlPath(path));
+    }
+
+    function teamUrlPath(path) {
+        return `/${currentTeam.name}/${id}/teamtest/${path}`;
+    }
+
+    return (
+        <Switch>
+            <Route path={`${teamUrlPath('subpath')}`}>
+                <h3>{'Hello, Subpath Component!'}</h3>
+            </Route>
+            <Route path={`${teamUrlPath('subpath-with-query')}`}>
```

Comment:
We can get rid of the template string here:
```suggestion
            <Route path={teamUrlPath('subpath')}>
                <h3>{'Hello, Subpath Component!'}</h3>
            </Route>
            <Route path={teamUrlPath('subpath-with-query')}>
```